packages/pangea-node-sdk/src/services/authn/user/index.ts

```diff
@@ -118,7 +122,10 @@ export default class User extends BaseService {
    * ```
    */
   update(
-    request: AuthN.User.Update.EmailRequest | AuthN.User.Update.IDRequest
```

Comment:
Same that with `request: AuthN.User.Authenticators.Delete.Request`?

Reply:
Looks like we won't be doing that.